client/verta/tests/registry/model_version/test_crud.py

```diff
@@ -193,10 +193,9 @@ def test_creation_from_run(self, registered_model, experiment_run, lock_level):
         assert model_ver._msg.lock_level == lock_level._as_proto()
```

Comment:
If possible, I do not think we should be removing any of the `pytest.raises(requests.HTTPError, match="^403")` checks in this file. They seem important to make sure non-admins cannot change lock level. Is it possible to keep them, or bring them back after we implement workspace creation?

(by the way: when tests run in Jenkins, `client` is admin and `client_2` is non-admin)

Reply:
I think workspace creation is needed for this test, I couldn't make it work with a default workspace. So I can add these checks again after workspace creation implemented in client

Reply:
reverted here: https://github.com/VertaAI/modeldb/pull/3439/files